File: app/src/main/java/com/easynfc/data/Text.java
package com.easynfc.data;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

/**
 * Created by pablorojas on 13/3/18.
 */

@Entity(tableName = "text")
public class Text {

    @PrimaryKey(autoGenerate = false)
    private long timeStamp;

    @ColumnInfo(name = "content")
    private String content;

    public Text(long timeStamp, String content) {
        this.timeStamp = timeStamp;
        this.content = content;
    }

    public long getTimeStamp() {
        return timeStamp;
    }

    public void setTimeStamp(long timeStamp) {
        this.timeStamp = timeStamp;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: app/src/main/java/com/easynfc/data/model/Text.kt
package com.easynfc.data.model

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.vipera.onepay.util.AppConstants

@Entity(tableName = "text")
data class Text(override var content: String, override var date: String, override var type: String = AppConstants.TYPE_TEXT) : BaseTag() {
    @PrimaryKey(autoGenerate = true)
    override var id: Int = 0
}